### DISCUSSION TITLE
Unable to see crashes using Crashlytics with Carthage

### DISCUSSION THREAD

**Discussion by RushanB** [firebase/firebase-ios-sdk#8712] [Q&A] [answered]
I followed the instructions found here https://github.com/firebase/firebase-ios-sdk/blob/master/Carthage.md#carthage-usage but tweaked them as my app is using custom naming for the info plist files for each environment ex:

GoogleService-Info-user-development.plist
GoogleService-Info-user-staging.plist
GoogleService-Info-user-production.plist

This is how I am attaching my plist to Firebase on `func application(_ application: UIApplication, willFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool`.

![D0D6028E-09FF-4FC6-B5A1-C7F6CFA99AF8](https://user-images.githubusercontent.com/16970644/134923341-151c2415-b5df-4289-ba5c-2aee1ee02768.png)

I have attached my script for crashlytics and it does not give any errors on app run. But I am unable to see any crashes in the firebase console.

![crashlytics-script](https://user-images.githubusercontent.com/16970644/134923115-ab77e80f-3c5e-4717-ba58-432be1d321c3.png)

**Comment by paulb777** [comment]
I would recommend looking at similar issues in the Issue Tracker for ideas. You may need to file a ticket with [Firebase Support](https://firebase.google.com/support) to look at your particular app.

**Comment by RushanB** [comment, marked as answer]
You can close this ticket, I found a solution and notified firebase support about it :)
